mentors/python/f.json

```diff
@@ -15,5 +15,14 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I love Python. I'm from Portugal. Started learning by myself a few years ago. Love to teach so that way I assure myself that I fully understand. Currently studying Software Engineering. I'm awesome :)"
+  },
+
```

Comment:
```suggestion
```
I know that line 10 is also blank, but it shouldn't be either.

Reply:
Right, my bad. I removed both blank lines.